Dockerfile

```diff
@@ -23,6 +23,22 @@ RUN npm install react-scripts@1.1.1 -g --silent
 # Copy everything to the working directory
 COPY . /usr/src/app
 
+# # Copy .env file and shell script to container
+COPY ./init.sh .
+COPY .env .
+
+# Make our shell script executable
+RUN chmod +x init.sh
+
+# Run the bash script
+RUN /bin/bash -c ./init.sh
```

Comment:
The whole point of having the bash script is to run it at runtime in the CMD docker command (see the very bottom) and not during the build. We want to change environment properties AT RUNTIME using a pre-built image.

Reply:
Let me work on that

Reply:
I have fixed the issue please review.